build.gradle

```
@@ -181,17 +190,29 @@ tasks.register('testNaming') {
 }
 
 enum Modifiers {
-	DEV_MODE, GEN_DOCS, DEBUG, PROFILE, JUNIT
+	DEV_MODE, GEN_DOCS, GEN_RELEASE_DOCS, DEBUG, PROFILE, JUNIT
```

I think a better name would be just `RELEASE_DOCS` otherwise it's confusing between that and GEN_DOCS
```suggestion
	DEV_MODE, GEN_DOCS, RELEASE_DOCS, DEBUG, PROFILE, JUNIT
```
Other places require update, don't commit suggestion.